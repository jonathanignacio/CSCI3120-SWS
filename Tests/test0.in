8080
0.0 0.0 sws.c