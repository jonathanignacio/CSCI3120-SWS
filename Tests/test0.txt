test0
Group 11

A very basic test, just to see if it can connect to one
thing with no delays or pauses.

8080
0.0 0.0 sws.c

