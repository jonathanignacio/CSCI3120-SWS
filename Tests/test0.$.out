HTTP/1.1 200 OK
Request 1 connected for Client
Request 1 completed